test/BUILD

```diff
@@ -157,6 +164,13 @@ scala_library(
            glob(["src/main/scala/scala/test/mix_java_scala/*.java"]),
     jvm_flags = ["-Xms1G", "-Xmx4G"],
 )
+#Mix java scala
+scala_library(
+    name = "MixJavaScalaLibWithJvmFlags",
```

Comment:
oh sorry, I think the "existing" one got it from #104 which you rebased on top off.
Any objection to removing this target?